tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyException.java

```diff
@@ -36,6 +36,12 @@ public void addDataTo(SurveyJSONWrapper r) throws JSONException {
         }
     }
 
+    /**
+     * Seems to not be used.
+     * @param err_data
+     * @return
+     */
+    @Deprecated
     public SurveyException setErrData(JSONObject err_data) {
         this.err_data = err_data;
```

Comment:
It's not used -- even in .jsp! I searched. So, better to delete it than deprecate it

Reply:
done